src/lexer.rs

```diff
@@ -85,6 +85,10 @@ impl<'f, T> Spanned<'f, T> {
             data: f(&self.data),
         }
     }
+
+    pub fn get_data(&self) -> &T {
+        &self.data
+    }
```

Comment:
Spanned implements `Deref`, so this is unnecassary. Instead of doing `spanned.get_data()`, you can just write `*spanned` (Btw, `data` is a public member anyway, so this getter is double-unnecesarry)

Reply:
I cannot find it. Seems to be resolved.